doc/chrome-webstore-permissions-descriptions.md

```diff
@@ -0,0 +1,29 @@
+# Privacy tab in chrome web store:
+-----------------------------------
+
+## Permissions:
+
+#### Privacy -
+Privacy Badger needs access to the privacy API so that it can restrict the webRTCIPHandlingPolicy so that none of the user's local address information is exposed, as well as if they're in an incognito window, it will override and disable the alternateErrorPagesEnabled and hyperlinkAuditingEnabled settings.
```

Comment:
>as well as if they're in an incognito window

Let's remove this and start a new sentence instead.

Reply:
The high-level gist is that we change several (two at the moment) privacy-related defaults to better protect users' privacy.

We also let users select a stricter privacy setting for WebRTC.

Reply:
I'm not sure at this point if the description should be a high level gist of why access to the API is needed, or if its a more granular look at each specific use case. If it's one or the other, should it then be consistent across all the different descriptions?

Reply:
Each description should make clear the why of us needing the permission in question. If you only say we set alternateErrorPagesEnabled, it's not clear what this Chrome setting has to do with Privacy Badger. **Why** do we set alternateErrorPagesEnabled? To better protect the privacy of our users, etc.